diff --git a/jupyterhub_config.py b/jupyterhub_config.py
index a71e21e..5233229 100755
--- a/jupyterhub_config.py
+++ b/jupyterhub_config.py
@@ -235,14 +235,14 @@ c.JupyterHub.db_url = 'postgresql://postgres:{password}@{host}/{db}'.format(
 )
 
 # https://github.com/jupyterhub/jupyterhub-idle-culler
-c.JupyterHub.services = [
-    {
-        "name": "jupyterhub-idle-culler-service",
-        "command": [
-            sys.executable,
-            "-m", "jupyterhub_idle_culler",
-            "--timeout=3600",
-        ],
-        "admin": True,
-    }
-]
+# c.JupyterHub.services = [
+#     {
+#         "name": "jupyterhub-idle-culler-service",
+#         "command": [
+#             sys.executable,
+#             "-m", "jupyterhub_idle_culler",
+#             "--timeout=3600",
+#         ],
+#         "admin": True,
+#     }
+# ]
